app/services/statements/query.rb

```diff
@@ -1,13 +1,21 @@
 module Statements
   class Query
-    def initialize(lead_provider:)
+    def initialize(lead_provider:, cohort_start_years: nil, updated_since: nil)
       @lead_provider = lead_provider
+      @cohort_start_years = cohort_start_years&.split(",")
+      @updated_since = updated_since
     end
 
     def statements
-      Statement
-        .includes(:cohort)
-        .where(lead_provider:)
+      scope = Statement
+                .includes(:cohort)
+                .where(lead_provider:)
```

Comment:
This ensures all queries need to specify a lead provider and won't really work for admin stuff where we might want to list all the statements.

Reply:
I think when it comes to reconciliation of services we can revisit this and amend as required, happy to keep as is for now